exposed-version-catalog/build.gradle.kts

```diff
@@ -0,0 +1,62 @@
+import org.jetbrains.exposed.gradle.configureMavenCentralMetadata
+import org.jetbrains.exposed.gradle.signPublicationIfKeyPresent
+
+plugins {
+    `version-catalog`
+    `maven-publish`
+    signing
+}
+
+group = "org.jetbrains.exposed"
+
+catalog {
+    versionCatalog {
+        val version: String by rootProject
+        
+        from(files("${rootProject.projectDir}/gradle/libs.versions.toml"))
```

Comment:
@jetbrains-junie Remove this line, we do not want to expose our own project dependencies to the consumer.